docs/source/metrics.rst

```diff
@@ -1,6 +1,42 @@
 ignite.metrics
 ==============
 
+Metrics provide a way to compute various quantities of interest in an online
+fashion without having to store the entire output history of a model.
+
+In practice user needs to attach metric instance to engine. Metric value is
+computed using engine's output:
+
+    .. code-block:: python
+
+        def process_function(engine, batch):
+            # ...
+            return y_pred, y
+
+        engine = Engine(process_function)
+        metric = CategoricalAccuracy()
+        metric.attach(engine, "accuracy")
+
+If engine's output is not in the format `y_pred, y`, user can use `output_transform`:
```

Comment:
If **the** engine's output is not in the format `y_pred, y`, **the** user can use  **the** `output_transform` **argument to transform it**: